common/lib/xmodule/xmodule/modulestore/tests/utils.py

```diff
@@ -72,6 +73,27 @@ def mock_tab_from_json(tab_dict):
     return tab_dict
 
 
+def add_temp_files_from_dict(file_dict, dir):
+    """
+    Takes in a dict formatted as: { file_name: content }, and adds files to directory
+    """
+    for file_name in file_dict:
+        with open("{}/{}".format(dir, file_name), "w") as opened_file:
```

Comment:
We should get in the habit of using `io.open()`, since the `open` builtin doesn't exist under Python 3.